include/clangmetatool/collectors/find_calls.h

```diff
@@ -0,0 +1,73 @@
+#ifndef INCLUDED_CLANGMETATOOL_COLLECTORS_FIND_CALLS_H
+#define INCLUDED_CLANGMETATOOL_COLLECTORS_FIND_CALLS_H
+
+#include <clang/Frontend/CompilerInstance.h>
+#include <clang/ASTMatchers/ASTMatchFinder.h>
+
+#include <clangmetatool/collectors/find_calls_data.h>
+
+namespace clangmetatool {
+  namespace collectors {
+
+    /**
+     * forward declaration to implementation details of the
+     * collector.
+     */
+    class FindCallsImpl;
+
+    /**
+     * Find Calls data collector. Collects the caller and references
+     * to the specified function and one of its arguments.
+     */
+    class FindCalls {
+    private:
+
+      /**
+       * Pointer to Implementation
+       */
+      FindCallsImpl* impl;
+
+    public:
+
+      /**
+       * Explicit constructor, to allow for implementation details:
+       *    - ci is a pointer to an instance of the clang compiler
+       *    - f is a pointer to an instance of the MatchFinder class
+       *    - n is a string value of the function name to match on
+       *    - a is an integer value of the argument position to resolve
+       */
+      FindCalls( clang::CompilerInstance          *ci,
+                 clang::ast_matchers::MatchFinder *f,
+                 std::string                       n,
+                 unsigned int                      a  );
```

Comment:
the support for `a` was dropped, right? so we should remove this parameter (and the reference from the doc)